partiql-planner/build.gradle.kts

```
@@ -77,3 +77,54 @@ publish {
     name = "PartiQL Planner"
     description = "PartiQL's Experimental Planner."
 }
+
+// Generate IR
```

nit: could make more clear this is for the internal IR

```suggestion
// Generate internal IR
```

---

ack